basyx.aasrepository/basyx.aasrepository.component/src/main/java/org/eclipse/digitaltwin/basyx/aasrepository/component/AasRepositoryConfiguration.java

```diff
@@ -48,6 +48,7 @@
  */
 @Configuration
 public class AasRepositoryConfiguration {
+	@Primary
```

Comment:
Why is this primary needed here? If there's another AasRepository bean available, then this is a design flaw.